assets/src/js/my-orders.js

```diff
@@ -0,0 +1,64 @@
+(function($){
```

Comment:
What's wrong with this file? There are alignment issues. We need a tab or four spaces for indentation but seems like two spaces are used for indentation in this file.